include/boost/variant/get.hpp

```diff
@@ -243,6 +263,30 @@ strict_get(
     return relaxed_get<U>(operand);
 }
 
+#ifndef BOOST_NO_CXX11_RVALUE_REFERENCES
+template <typename U, BOOST_VARIANT_ENUM_PARAMS(typename T) >
+inline
+    U&&
+strict_get(
+      boost::variant< BOOST_VARIANT_ENUM_PARAMS(T) >&& operand
+      BOOST_VARIANT_AUX_GET_EXPLICIT_TEMPLATE_TYPE(U)
+    )
+{
+    BOOST_STATIC_ASSERT_MSG(
+        (mpl::not_< boost::is_lvalue_reference<U> >::value),
```

Comment:
Prefer `!boost::is_lvalue_reference<U>::value` (it is simpler and theoretically faster to compile because it instantiates less templates).